x-pack/plugins/maps/public/components/global_time_checkbox.tsx

```diff
@@ -21,13 +21,21 @@ export function GlobalTimeCheckbox({ applyGlobalTime, label, setApplyGlobalTime
 
   return (
     <EuiFormRow display="columnCompressedSwitch">
-      <EuiSwitch
-        label={label}
-        checked={applyGlobalTime}
-        onChange={onApplyGlobalTimeChange}
-        data-test-subj="mapLayerPanelApplyGlobalTimeCheckbox"
-        compressed
-      />
+      <EuiToolTip
+        position="top"
+        content={i18n.translate('xpack.maps.filterEditor.applyGlobalTimeHelp', {
+          defaultMessage:
+            'When enabled, time filter from global search bar is applied to this layer"',
```

Comment:
GlobalTimeCheckbox is used in two places, Layer filtering configuration and Term join configuration. The message needs to be usable in both places and as such should not contain the phrase "this layer".

How about just "When enabled, results narrowed by global time filter"